test/unit/index.js

```diff
@@ -1,41 +1,75 @@
-require('./api/ws/workers/connectionsTable');
-require('./api/ws/workers/peersUpdateRules');
-require('./api/ws/workers/rules');
-require('./api/ws/workers/slaveToMasterSender');
-
-require('./helpers/ed');
-require('./helpers/jobs-queue');
-require('./helpers/peersManager');
-require('./helpers/pg-notify');
-require('./helpers/request-limiter');
-require('./helpers/RPC');
-require('./helpers/slots');
-require('./helpers/wsApi');
-require('./helpers/z_schema');
-
-require('./logic/account');
-require('./logic/delegate');
-require('./logic/peer');
-require('./logic/peers');
-require('./logic/multisignature');
-require('./logic/transaction');
-require('./logic/transactionPool');
-require('./logic/transfer');
-require('./logic/signature.js');
-require('./logic/vote');
-
-require('./modules/blocks/process');
-require('./modules/blocks/verify');
-require('./modules/accounts');
-require('./modules/app');
-require('./modules/blocks');
-require('./modules/cache');
-require('./modules/delegates');
-require('./modules/loader');
-require('./modules/peers');
-require('./modules/transactions');
-
-require('./sql/delegatesList');
-require('./sql/rounds');
-
-require('./schema/delegates');
+'use strict';
+
+var cluster = require('cluster');
+var child_process = require('child_process');
+
+var paths = [
+	'./api/ws/workers/connectionsTable.js',
+	'./api/ws/workers/peersUpdateRules.js',
+	'./api/ws/workers/rules.js',
+	'./api/ws/workers/slaveToMasterSender.js',
+
+	'./helpers/ed.js',
+	'./helpers/jobs-queue.js',
+	'./helpers/peersManager.js',
+	'./helpers/request-limiter.js',
+	'./helpers/RPC.js',
+	'./helpers/slots.js',
+	'./helpers/wsApi.js',
+	'./helpers/z_schema.js',
+
+	'./logic/account.js',
+	'./logic/delegate.js',
+	'./logic/peer.js',
+	'./logic/peers.js',
+	'./logic/multisignature.js',
+	'./logic/transaction.js',
+	'./logic/transactionPool',
```

Comment:
This file does not have extension `.js`. Does it matter?

Reply:
For consistency, better to keep as is, with extension.